gcsfs/tests/test_core.py

```diff
@@ -59,6 +59,13 @@ def test_info(token_restore):
         assert gcs.info(a) == gcs.ls(a, detail=True)[0]
 
 
+@my_vcr.use_cassette(match=['all'])
+def test_info_directory(token_restore):
```

Comment:
You need to set the various env variables in the test settings module and run with recording on (will produce a YAML file) before tests will pass.

Reply:
I think that this won't work regardless

```python
In [1]: from gcsfs import GCSFileSystem

In [2]: gcs = GCSFileSystem()

In [3]: gcs.ls('pangeo-data')
Out[3]: 
['pangeo-data/newman-met-ensemble/',
 'pangeo-data/newmann-met-ensemble-netcdf/',
 'pangeo-data/test997/',
 'pangeo-data/test998/',
 'pangeo-data/test999/']

In [4]: gcs.info('pangeo-data/test997/')
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-4-210f02b52f31> in <module>()
----> 1 gcs.info('pangeo-data/test997/')

/home/mrocklin/Software/anaconda/lib/python3.6/site-packages/gcsfs/core.py in info(self, path)
    468             return out[0]
    469         else:
--> 470             raise FileNotFoundError(path)
    471 
    472     def url(self, path):

FileNotFoundError: pangeo-data/test997/
```